integration-tests/it-common/src/main/java/fr/pilato/elasticsearch/crawler/fs/test/integration/elasticsearch/FsCrawlerTestTikaConfigPathIT.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to David Pilato (the "Author") under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Author licenses this
+ * file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package fr.pilato.elasticsearch.crawler.fs.test.integration.elasticsearch;
+
+import fr.pilato.elasticsearch.crawler.fs.client.ESMatchQuery;
+import fr.pilato.elasticsearch.crawler.fs.client.ESSearchRequest;
+import fr.pilato.elasticsearch.crawler.fs.settings.Fs;
+import fr.pilato.elasticsearch.crawler.fs.test.integration.AbstractFsCrawlerITCase;
+import fr.pilato.elasticsearch.crawler.fs.tika.TikaInstance;
+import java.io.File;
+import java.io.InputStream;
+import org.apache.commons.io.FileUtils;
+import org.junit.Test;
+
+/**
+ * Test tika config path crawler setting
+ */
+public class FsCrawlerTestTikaConfigPathIT extends AbstractFsCrawlerITCase {
+
+  @Test
+  public void test_tika_config_path() throws Exception {
+    TikaInstance.reloadTika();
+    InputStream tikaConfigIS = getClass().getResourceAsStream("/documents/tikaConfig.xml");
+    File tikaConfigFile = File.createTempFile("tikaConfigTestFile", ".xml");

```

Comment:
I'd prefer to use `rootTmpDir` which is generated automatically as a tmp folder when we start tests.
Could you use it instead and write the file wherever you want in it?

```java
Path testTikaConfig = rootTmpDir.resolve("tika-config");
if (Files.notExists(testTikaConfig)) {
  Files.createDirectory(testTikaConfig);
}

// Copy the file in testTikaConfig
```

That being said, I don't understand why you need to copy the file instead of using it directly...
Specifically if you move the config file in the dedicated test folder, like `samples/test_tika_config_path/`, it will be automatically copied to the tmp folder when the test starts by `AbstractITCase#copyTestResources()`.

The URL of this folder is `currentTestResourceDir`. 
You should get the URL of the config file with `currentTestResourceDir.resolve("tikaConfig.xml");`

Reply:
Putting in samples/test_tika_config_path made it a lot nicer! Thank you for that suggestion